src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExporterExtensions.cs

```diff
@@ -63,6 +63,7 @@ public static class DynatraceMetricsExporterExtensions
 
 			var metricExporter = new DynatraceMetricsExporter(options, logger);
 			var metricReader = new PeriodicExportingMetricReader(metricExporter, options.MetricExportIntervalMilliseconds);
+			metricReader.TemporalityPreference = MetricReaderTemporalityPreference.Delta;
```

Comment:
So, by setting this, I guess the SDK uses the Func that has the mapping for instruments that use Delta right? As in this table https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md

Reply:
Yes, it uses [this mapper function](https://github.com/open-telemetry/opentelemetry-dotnet/blob/33125ac3bc3c5aa96a6a203adf844ebfe912a05b/src/OpenTelemetry/Metrics/MetricReader.cs#L36-L58). :slightly_smiling_face: